packages/catalog-model/src/entity/Entity.ts

```diff
@@ -48,7 +48,7 @@ export type Entity = {
  * @see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta
  * @see https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/
  */
-export type EntityMeta = {
+export type EntityMeta = Record<string, any> & {
```

Comment:
technically unrelated change but was needed by the tests - this lets TS realise that metadata is extensible (can have more fields than specced here)